modules/collect/src/main/java/com/opengamma/strata/collect/result/ValueWithFailures.java

```diff
@@ -87,6 +89,27 @@
   }
 
   //-------------------------------------------------------------------------
+
+  /**
+   * Returns a {@link Collector} which performs a reduction of its
+   * {@link ValueWithFailures} input elements under a specified {@link BinaryOperator} using the
+   * provided identity.
+   *
+   * @param identityValue the identity value
+   * @param operator the operator used for the reduction.
+   * @param <T> the type of the success value in the {@link ValueWithFailures}
```

Comment:
Strata puts `<T>` parameter first.
Strata has two spaces not one after parameter name

```
 * @param <T> the type of the success value in the {@link ValueWithFailures}
 * @param identityValue  the identity value
 * @param operator  the operator used for the reduction.
```